avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -883,9 +883,15 @@ output controlPlaneFQDN string = enablePrivateCluster
 @description('The principal ID of the system assigned identity.')
 output systemAssignedMIPrincipalId string = managedCluster.?identity.?principalId ?? ''
 
+@description('The Client ID of the AKS identity.')
+output kubeletidentityClientId string = managedCluster.properties.?identityProfile.?kubeletidentity.?clientId ?? ''
+
 @description('The Object ID of the AKS identity.')
 output kubeletidentityObjectId string = managedCluster.properties.?identityProfile.?kubeletidentity.?objectId ?? ''
 
+@description('The Resource ID of the AKS identity.')
+output kubeletidentityResourceId string = managedCluster.properties.?identityProfile.?kubeletidentity.?resourceId ?? ''
```

Comment:
Are you sure it's `.resourceId` and not `.id` like resources usually handle this property?

Reply:
Thanks for your comments! I checked the [doc](https://learn.microsoft.com/en-us/azure/templates/microsoft.containerservice/2023-07-02-preview/managedclusters?pivots=deployment-language-bicep) and it was stored directly with `resourceId`.
![image](https://github.com/Azure/bicep-registry-modules/assets/81678720/010541fd-ae78-41a2-b1f0-38ab3eaa4433)

Reply:
Right, but that's the input, correct, not the output.

Anyhow, if you get the change to deploy the module via your fork (mind you, you can unselect the static tests to go straight to the deployment tests - just please first apply the namePrefix fix I called out in the other comment), it would be great if you could do the following check
- While the test for e.g. the max test is running, go to your subscription
- Look for the deployment on the subscription
- Once it completes, and before the resource removal kicks in, navigate to the outputs of the deployment and check if the new output shows up (which it definitely should) and if it has a value. 
![image](https://github.com/Azure/bicep-registry-modules/assets/5365358/f111c899-0883-452e-bdfa-dc49cc343ba3)

If it is empty, I'd suggest to try `id` instead. Sorry for adding the extra effort, but I want to make sure this output definitely returns what you expect before we publish the module with them 💪

Reply:
Looks great! Thanks for your patience.

I did what you said and the `resourceId` can be used normally. Thank you very much!

![Screenshot 2024-04-02 163300](https://github.com/Azure/bicep-registry-modules/assets/81678720/fea09ab4-782a-46b0-b727-48208ec8be04)

Reply:
Glad to hear, thanks for the due dilligence 💪 . Resolving...